src/commands/generate_buildpack_matrix/command.rs

```diff
@@ -71,42 +82,267 @@ pub(crate) fn execute(args: &GenerateBuildpackMatrixArgs) -> Result<()> {
 
     actions::set_output("version", version).map_err(Error::SetActionOutput)?;
 
+    let rust_triples = buildpacks
+        .iter()
+        .flat_map(read_buildpack_targets)
+        .filter_map(|t| rust_triple(&t).ok())
+        .collect::<HashSet<String>>();
+
+    actions::set_output(
+        "rust_triples",
+        serde_json::to_string(&rust_triples).map_err(Error::SerializingJson)?,
+    )
+    .map_err(Error::SetActionOutput)?;
+
```

Comment:
@joshwlewis We should provide [summaries](https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#adding-a-job-summary) for these outputs.  E.g.; pretty-print the json into a markdown code block or format it into a table. This would be useful for communicating what's being processed (especially now that multiple targets are involved).

It would be easy to follow a similar pattern as `actions::set_output` with a `actions::set_summary` function that writes to `$GITHUB_STEP_SUMMARY`.

Reply:
Done!